README.md

```diff
@@ -44,15 +61,38 @@ The default location for locally-installed providers is one of the following, de
 
 Place the provider into the plugins directory, for example:
 ```bash
-version=0.2.9
+version=0.0.0-master
 arch=linux_amd64
 terraform_harvester_provider_bin=./bin/terraform-provider-harvester
-
-terraform_harvester_provider_dir="${HOME}/.terraform.d/plugins/registry.terraform.io/harvester/harvester/${version}/${arch}/"
+terraform_harvester_providers_dir="${HOME}/.terraform.d/plugins/registry.terraform.io/harvester/harvester
+terraform_harvester_provider_dir="${terraform_harvester_providers_dir}/${version}/${arch}/"
 mkdir -p "${terraform_harvester_provider_dir}"
 cp ${terraform_harvester_provider_bin} "${terraform_harvester_provider_dir}/terraform-provider-harvester_v${version}"}
 ```
 
+After placing it into your plugins directory,  copy and paste this code into your Terraform configuration. Then, run `terraform init` to initialize it.
+```hcl
+terraform {
+  required_providers {
+    harvester = {
+      source = "harvester/harvester"
+      version = "0.0.0-master"
+    }
+  }
+}
+
+provider "harvester" {
+  # Configuration options
+}
+```
+
+After testing, If you want to use the harvester provider from registry.terraform.io, you need to delete the local terraform provider harvester dir by:
```

Comment:
If you want to switch back to the Harvester provider from `registry.terraform.io` after testing, you need to delete the local build manually via: